build.gradle.kts

```diff
@@ -88,6 +90,15 @@ tasks.register("allDependencies") {
     }
 }
 
+open class OrtServerPrintTask : PrintTask({ "" }, "Prints the current project version") {
```

Comment:
`VersionPrintTask` would be a better more specific name.

Reply:
@MarcelBochtler, if you care to change it, I'd be fine with the same name in ORT.